src/asf_tools/flood_map.py

```diff
@@ -222,30 +224,102 @@ def make_flood_map(out_raster: Union[str, Path],  vv_raster: Union[str, Path],
 
 
-def main():
+def _get_cli(interface: Literal['hyp3', 'main']) -> argparse.ArgumentParser:
     parser = argparse.ArgumentParser(
         description=__doc__,
         formatter_class=argparse.ArgumentDefaultsHelpFormatter
     )
-    parser.add_argument('out_raster',
-                        help='File flood depth map will be saved to.')
-    parser.add_argument('vv_raster',
-                        help='Sentinel-1 RTC GeoTIFF raster, in power scale, with VV polarization')
-    parser.add_argument('water_extent_map',
-                        help='Hyp3-Generated water extent raster file.')
-    parser.add_argument('hand_raster',
-                        help='Height Above Nearest Drainage (HAND) GeoTIFF aligned to the RTC rasters. '
-                             'If not specified, HAND data will be extracted from a Copernicus GLO-30 DEM based HAND.')
-
-    parser.add_argument('--estimator', type=str, default='iterative', choices=['iterative', 'logstat', 'nmad', 'numpy'],
+
+    available_estimators = ['iterative', 'logstat', 'nmad', 'numpy']
+    if interface == 'hyp3':
+        parser.add_argument('--bucket')
+        parser.add_argument('--bucket-prefix', default='')
+        parser.add_argument('--wm-raster',
+                            help='Water map GeoTIFF raster, with suffix `_WM.tif`.')
+        available_estimators += ['None']
+    elif interface == 'main':
+        parser.add_argument('out_raster',
+                            help='File flood depth map will be saved to.')
+        parser.add_argument('vv_raster',
+                            help='Sentinel-1 RTC GeoTIFF raster, in power scale, with VV polarization')
+        parser.add_argument('water_extent_map',
+                            help='Hyp3-Generated water extent raster file.')
+        parser.add_argument('hand_raster',
+                            help='Height Above Nearest Drainage (HAND) GeoTIFF aligned to the RTC rasters. '
+                                 'If not specified, HAND data will be extracted from the GLO-30 HAND.')
+    else:
+        raise NotImplementedError(f'Unknown interface: {interface}')
+
+    parser.add_argument('--estimator', type=str, default='iterative', choices=available_estimators,
                         help='Flood depth estimation approach.')
     parser.add_argument('--water-level-sigma', type=float, default=3.,
                         help='Estimate max water height for each object.')
     parser.add_argument('--known-water-threshold', type=float, default=30.,
                         help='Threshold for extracting known water area in percent')
-    parser.add_argument('--iterative-bounds', type=int, nargs=2, default=[0, 15], help='.')
+
+    if interface == 'hyp3':
+        parser.add_argument('--iterative-min', type=int, default=0)
+        parser.add_argument('--iterative-max', type=int, default=15)
+    elif interface == 'main':
+        # FIXME: why `help='.'`?
```

Comment:
I traced it back to here: https://github.com/ASFHyP3/asf-tools/commit/8fda37c1b9d1e3c434f4ae56e08f26f33ff1a421#diff-a9c485f9ad0ee40ddc1a28e0af7cc831500161e44d10607e47bcfeacf98876bdR224

@jacquelynsmale Was it just a placeholder help text?